test.js

```diff
@@ -22,12 +22,25 @@ const initialWorkingDir = process.cwd();
 const mkCommand =
   (cmd) =>
   (args, options = {}) => {
+    let envVars = { SLS_DEBUG: 't' };
+    if (cmd == 'sls') {
+      // convert deprecated CLI arguments to env vars
+      const argsCopy = [...args];
```

Comment:
While this is a working solution, I think it would be better to explicitly pass these options as `env` to `sls` created with `mkCommand` utility. What do you think? For example, invocation could look like this:

```
sls(['package'], {env: {DOCKERIZE_PIP: true, SLIM: true}});
```

instead of 

```
sls(['--dockerizePip=true', '--slim=true', 'package']);
```

Reply:
I totally agree, but I'd need to spend time on manually rewriting each function... ;/
Maybe I'll come up with a script to parse these lines for me if you insist on this - if you don't, then I think it's gonna be good to merge after I push the lint correction.

Reply:
Hey @martinezpl - I think we will need to address it anyway - if you won't have time to address it, I can give it a try, I'd rather not merge in code that needs to be changed soon. Let me know 💯

Reply:
Done, turned out to be a pretty quick task with a bit of regex.